content/learn/links.toml

```diff
@@ -5,6 +5,13 @@ image = "/assets/book.svg"
 image_alt = "Bevy book"
 description = "Learn how to use Bevy using this step-by-step guide. If you are new to Bevy, you should probably start here."
 
```

Comment:
should the book link above this line be hidden? it's 404ing

Reply:
That is removed in #879 which has been merged already.